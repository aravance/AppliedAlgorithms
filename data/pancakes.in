A
a
B
b
C
c
D
d
E
e
F
f
AA
aa
BB
bb
CC
cc
DD
dd
EE
ee
FF
ff
AB
Ab
aB
ab
BA
Ba
bA
ba
AC
AF
Af
aF
af
FA
Fa
fA
fa
AAA
aaa
FFF
fff
ABF
fba
ABC
ABc
AbC
Abc
aBC
aBc
abC
abc
ACB
ACb
AcB
Acb
aCB
aCb
acB
acb
BAC
BAc
BaC
Bac
bAC
bAc
baC
bac
BCA
BCa
BcA
Bca
bCA
bCa
bcA
bca
CAB
CAb
CaB
Cab
cAB
cAb
caB
cab
CBA
CBa
CbA
Cba
cBA
cBa
cbA
cba
fca
ABCD
ABCDE
ABCDEF
abc
abcd
abcde
abcdef
fedcba
fffABC
ffccaa
FFCCAA
0


