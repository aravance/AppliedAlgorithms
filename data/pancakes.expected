0
1
0
1
0
1
0
1
0
1
0
1
0
1
0
1
0
1
0
1
0
1
0
1
0
1
3
4
3
2
2
1
0
0
1
3
4
3
2
2
1
0
1
0
1
0
1
0
1
3
4
3
4
6
6
3
2
2
1
5
5
5
4
4
3
4
4
4
4
3
4
3
2
4
3
4
3
4
4
3
4
4
4
2
3
3
4
5
5
3
2
5
4
2
1
1
0
0
0
6
8
10
12
1
2
1
5
